src/tutorial/databases.md

```diff
@@ -2,29 +2,31 @@
 
 ## Working with Databases
 
-We just learned how to build a basic [CRUD](CRUD.md) application with an in-memory database. Now, we are going to step it up notch and work with a persistent database. Meaning your data will be saved even after you shut down your application.
+You just learned how to build a basic [CRUD](CRUD.md) application with an in-memory database. Now, you are going to step it up notch and work with a persistent database. Meaning your data will be saved even after you shut down your application.
 
- For this tutorial we will be using [SQLite database](https://www.sqlite.org/index.html) but, you may use on that works better for you. 
+ For this tutorial, you'll be using [SQLite database](https://www.sqlite.org/index.html) but, you may use one that works better for you.
 
 ### Learn checklist Three ✔️
-- Setup SQLite database
+
+- Set up SQLite database
 - Create a SQLite database
-- Perform SQLite CRUD operation from our todo api
+- Perform SQLite CRUD operation from your todo api
+
+## Set up SQLite database
 
-## Setup SQLite database
-**Setup SQLite Database using Entity Framework Core(EF Core)***
+**Set up SQLite Database using Entity Framework Core(EF Core)**
 
 **Install the following tools and packages**
 
 Using .NET CLI / Visual Studio package manager UI, install the following packages:
 
-**[SQLite EF Core Database Provider](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite/6.0.0-preview.5.21301.9)** : can access many different databases through plug-in libraries called [database providers](https://docs.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli). The package below is the SQLite database provider for EF Core.
+**[SQLite EF Core Database Provider](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite/6.0.0-preview.5.21301.9)**: can access many different databases through plug-in libraries called [database providers](https://docs.microsoft.com/ef/core/providers/?tabs=dotnet-core-cli). The following package is the SQLite database provider for EF Core.
```

Comment:
we're now in preview 7. should url and commands be updated?

Reply:
This should be RC1 for the next release .